sigs/sig-scalability/README.md

```diff
@@ -0,0 +1,19 @@
+# Argo SIG Scalability
+
+Argo SIG Scalability is focused on the scalability and performance of Argo CD project.
+
+The goals of this group are to create benchmarking tooling to objectively measure the scaliablity of the project, and improving [the "scaling up" experience](https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#scaling-up) for end users.
+
+## Meetings
+SIG Scalability meetings are held on the second and forth Wednesday of each month. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com).
+
+The group can also be found on [the CNCF Slack](https://slack.cncf.io/), in the `#argo-sig-scalability` channel.
```

Comment:
Another way to contact the SIG could be using [the `type:scalability` label](https://github.com/argoproj/argo-cd/pulls?q=is%3Apr+is%3Aopen+label%3Atype%3Ascalability) on PRs and issues.